docs/_articles/en/scanner-commands/add.md

```diff
@@ -24,24 +24,24 @@ $ sfdx scanner:rule:add -l <string> -p <array> [--json]
 ```
   
 ## Example
-Bundle custom PMD rules in JAR files. Be sure to adhere to PMD conventions, such as defining the custom rules in XML files under a ```/category``` directory.
+XPath-only custom PMD rules may be added as standalone XML files. Java-based rules must be bundled in JAR files. Be sure to adhere to PMD conventions, such as defining the custom rules in XML fils under a ```/category``` directory.
 
 See the [PMD documentation](https://pmd.github.io/latest/pmd_userdocs_extending_writing_pmd_rules.html) for information about writing rules. 
   
-This example shows how to specify two JAR files directly.
+This example shows how to specify two rule files directly.
```

Comment:
Can we say "multiple files" and not imply the count of rules? A jar could contain multiple rules and a standalone XML could contain multiple XPath rules.